app/controllers/todos_controller.rb

```diff
@@ -42,6 +41,7 @@ def create
         format.html { render :new }
         format.json { render json: @todo.errors, status: :unprocessable_entity }
       end
+      # TodoMailer.weekly_report(@todo).delivery_now
```

Comment:
is this comment required?

Reply:
sorry i'm working in this feature thats why the comment